src/Ocis.php

```diff
@@ -480,6 +481,64 @@ public function createDrive(
         );
     }
 
+    /**
+     * @param array<int,GroupOrder|GroupProperties|GroupExpandEntities> $enumList
+     * @return array<int,string>
+     */
+    private function getStringFromEnumClass(array $enumList): array
+    {
+        $valueList = [];
+        foreach ($enumList as $enum) {
+            $valueList[] = $enum->value;
+        }
+        return $valueList;
+    }
+
+    /**
+     * @param string $search
+     * @param GroupOrder[] $orderBy
+     * @param GroupProperties[] $select
+     * @param GroupExpandEntities[] $expand
+     *
+     * @throws \Exception
+     *
+     * @return Group[]
+     */
+    public function listAllGroups(string $search = "", $orderBy = [], $select = [], $expand = [])
```

Comment:
1. we might not list all groups, so lets call the function `listGroups`
2. how can orderBy an array? What is the advantage of that?
3. `$select` does not feel that needed, lets just return the default
4. `$expand` can only expand members, so we can have a bool called `expandMembers`